client/me/purchases/manage-purchase/index.jsx

```diff
@@ -646,10 +647,42 @@ class ManagePurchase extends Component {
 					</>
 				);
 			}
-
 			return description;
 		}
 
+		if ( isDIFMProduct( purchase ) ) {
+			const productDetails = productsList[ WPCOM_DIFM_LITE ];
+			const [ tier0 ] = productDetails.price_tier_list;
+			const { maximum_units: noOfBaselinePagesIncluded } = tier0;
+
+			const description = translate(
+				'Hire a professional to set up your %(noOfBaselinePagesIncluded)d page website.',
+				{
+					args: {
+						noOfBaselinePagesIncluded: noOfBaselinePagesIncluded,
+					},
+				}
+			);
+
+			if ( purchase.purchaseRenewalQuantity ) {
+				return (
+					<>
+						{ description }{ ' ' }
+						{ translate(
+							'This purchase includes %(numberOfPages)d extra page.',
+							'This purchase includes %(numberOfPages)d extra pages.',
+							{
+								count: purchase.purchaseRenewalQuantity,
```

Comment:
The `count` should also be `purchase.purchaseRenewalQuantity - noOfBaselinePagesIncluded`.